.github/workflows/release-1x-integration.yaml

```diff
@@ -0,0 +1,61 @@
+name: Audio Integration Workflow 1.x
+
+on:
+  schedule:
+    - cron: '30 * * * *'
+
+env:
+ SELENIUM_GRID_PROVIDER: saucelabs
+ CLOUD_WATCH_METRIC: false
+ TEST_TYPE: Github-Action
+ SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
+ SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+
+jobs:
+  integ-audio:
+    name: Audio Integration Test 1.x
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Create a Job ID
+        id: create-job-id
+        uses: filipstefansson/uuid-action@ce29ebbb0981ac2448c2e406e848bfaa30ddf04c
+      - name: Set JOB_ID Env Variable
+        run: echo "JOB_ID=${{ steps.create-job-id.outputs.uuid }}" >> $GITHUB_ENV
+      - name: Echo Job ID
+        run: echo "${{ steps.create-job-id.outputs.uuid }}"
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Setup Sauce Connect
+        uses: saucelabs/sauce-connect-action@v1
+        with:
+          username: ${{ secrets.SAUCE_USERNAME }}
+          accessKey: ${{ secrets.SAUCE_ACCESS_KEY }}
+          noSSLBumpDomains: all
+          tunnelIdentifier: ${{ steps.create-job-id.outputs.uuid }}
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          path: current
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          ref: release-1.x
+          path: prev
+      - name: Setup Node.js - 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
```

Comment:
Should this be 15.x?

Reply:
Did i understand that this one is testing 1.x ? I was looking for current version (which is 2.x), and last 2 versions, say 2.x-1 and 2.x-2

Reply:
Revise it to previous current version

Reply:
We should not use 15 here as it is already deprecated in favor of 16 but node 16 has issue with sass compiler

Reply:
So this will currently test the last version of latest major version only and we will extend it other versions later right as required?

Reply:
Yep